<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidgetStrategy</class>
 <widget class="QDockWidget" name="DockWidgetStrategy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Strategy</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Buy IV</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="pushButtonStopSG">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Sell IV</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxbuySpread">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QComboBox" name="tradeSymbolUnderlying">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Underlying</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Trade Symbol</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSellSpread">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="pushButtonReload">
         <property name="text">
          <string>Reload Symbols</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxSymbol">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="pushButtonStartSG">
         <property name="text">
          <string>Start/Modify</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxOrderLots">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Order lots</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="labelBidIv">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string> Bid IV</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="labelAskIv">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current IV</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="ivVal">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
